pkg/landscaper/jsonschema/loader.go

```diff
@@ -186,10 +190,30 @@ func (l *Loader) loadComponentDescriptorReference(refURL *url.URL) ([]byte, erro
 	}
 
 	var JSONSchemaBuf bytes.Buffer
-	_, err = blobResolver.Resolve(ctx, res, &JSONSchemaBuf)
+	info, err := blobResolver.Resolve(ctx, res, &JSONSchemaBuf)
 	if err != nil {
 		return nil, fmt.Errorf("unable to fetch jsonschema for '%s': %w", refURL.String(), err)
 	}
 
+	mt, err := mediatype.Parse(info.MediaType)
+	if err != nil {
+		return nil, fmt.Errorf("unable to parse media type %q: %w", info.MediaType, err)
+	}
+	if mt.Type != mediatype.JSONSchemaArtifactsMediaTypeV1 {
```

Comment:
With this change it seems that we would drop support for `JSONSchemaArtifactsMediaTypeV0_`.
Should this be added to the release notes?

Reply:
no its not dropped as `JSONSchemaArtifactsMediaTypeV0` is automatically converted to `JSONSchemaArtifactsMediaTypeV1` here: https://github.com/gardener/landscaper/blob/master/apis/mediatype/constants.go#L51

Reply:
👍